### PR TITLE
feat: receipt views

### PR DESCRIPTION
**Pull request by dexturr**
## Receipt views

- Withdrawal
- Order submission
- Order cancellation
- Order ammendment

Notes:

- Iceberg and pegged not supported
- Withdrawal amount not support
- No enrichment currently

Screenshot 2023-08-30 at 12 37 09
Screenshot 2023-08-30 at 12 36 09

### CODE REVIEW

**Comment by dalebennett1992** [comment]
Feedback as it stands...

**Order submission observations:**
weird visual anomolies caused by enums not being handled. If I set timeInForce to '2' we get an empty grey box
image
Same applies for the order type, we do not see a type if we set an enum value and not a string. (screenshot does not demonstrate this) --This issue will be addressed [here](https://github.com/vegaprotocol/vegawallet-browser/issues/496)

**Withdrawal Submission observations**
image
Looks ok bar the missing asset stuff which is coming later

**Order Cancellation**
image

**Order Amendment**
image

File: frontend/components/receipts/orders/submission/submission.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        transaction={{
+          orderSubmission: {
+            icebergOpts: {}
+          }
+        }}
+      />
+    )
+    expect(container).toBeEmptyDOMElement()
+  })
+  it('renders order table and badges', () => {
```
Comment (dalebennett1992) [inline frontend/components/receipts/orders/submission/submission.spec.tsx:30]: looks to me like we are missing quite a few assertions here to cover the below AC

File: frontend/components/receipts/orders/submission/submission.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    // 1114-RCPT-012 I can see the market id of the order I am submitting
+    // 1114-RCPT-013 I can see the direction of order I am submitting
+    // 1114-RCPT-014 I can see the type of the order I am submitting
+    // 1114-RCPT-015 I can see the reference of the order I am submitting
```
Comment (dexturr) [inline frontend/components/receipts/orders/submission/submission.spec.tsx:34]:
```suggestion
```

File: frontend/components/receipts/orders/amend/amend.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    const { container } = render(<Amendment transaction={{ orderAmendment: { pegged_offset: {} } }} />)
+    expect(container).toBeEmptyDOMElement()
+  })
+  it('should render nothing if the order has pegged reference set', () => {
+    const { container } = render(<Amendment transaction={{ orderAmendment: { pegged_reference: {} } }} />)
+    expect(container).toBeEmptyDOMElement()
+  })
+  it('should render badges and table data', () => {
+    // 1114-RCPT-019 I can see the order id of the order I am amending
+    // 1114-RCPT-020 I can see the market id relating to the order I am amending
```
Comment (dexturr) [inline frontend/components/receipts/orders/amend/amend.spec.tsx:16]:
```suggestion
```

**Pull request merged**